### PR TITLE
handled logs loading on query changes

### PR DESCRIPTION
**Pull request by Prashanth-Nelli**
- handled logs loading on query changes
- removed current behaviour of loading logs on page load.

### CODE REVIEW

File: src/components/Logs/Table/logs-table.js [hunk 1]
```diff
@@ -127,7 +127,11 @@ export const LogsTable = ({ logs, actions, variant }) => {
             return (
               <tr key={key} {...restHeaderGroupProps}>
                 {headerGroup.headers.map((column) => (
-                  <th key={column.id} {...column.getHeaderProps()}>
+                  <th
+                    key={column.id}
+                    {...column.getHeaderProps()}
+                    className="pt-2 pb-2"
```
Comment (ciju) [inline src/components/Logs/Table/logs-table.js:133]: `py-2`?

File: src/pages/logs-page.js [hunk 2]
```diff
@@ -44,15 +44,15 @@ export const logTypes = [
 ];
 
 export function LogsPage() {
-  const [logsIsLoading, setLogsIsLoading] = useState(true);
+  const [logsIsLoading, setLogsIsLoading] = useState();
```
Comment (ciju) [inline src/pages/logs-page.js:47]: Could be explicit about the loading states. E.g. `initial | loading | done`

File: src/pages/logs-page.js [hunk 3]
```diff
@@ -99,16 +99,17 @@ export function LogsPage() {
...
   }, []);
 
   useEffect(() => {
-    loadLogs();
-  }, [loadLogs, topologyId]);
+    const subscription = watch(() => {
+      loadLogs();
+    });
+    return () => subscription.unsubscribe();
+  }, [watch]);
```
Comment (ciju) [inline src/pages/logs-page.js:112]: We don't need to watch for `topologyId` here? Including `loadLogs` in should also be ok?

Comment (Prashanth-Nelli) [reply src/pages/logs-page.js:146]: we are watching all controls changes here so topologyId is also covered. loadlogs not required as it's reference won't change.

**Review by ciju — APPROVED** [review]

**Pull request merged**